lib/matplotlib/axes/_base.py

```diff
@@ -558,6 +558,12 @@ class _AxesBase(martist.Artist):
     dataLim: mtransforms.Bbox
     """The bounding `.Bbox` enclosing all data displayed in the Axes."""
 
+    spines: mspines.Spines
+    """
+    The `.Spines` container for the Axes' spines, i.e. the lines denoting the
+    data area boundaries.
```

Comment:
Maybe:

```suggestion
    The `.Spines` container for the Axes' spines, usually the lines that denote the x- and y-axis.
```

because the spines need not be on the data area boundaries (of course they need not be x or y axises either, but "usually" should be fine for that)?

Reply:
I slightly lean towards the present description. While not general, it's what the default is, and thus it's easier to understand. Possibly a more detailed description should go into the class docs?

Reply:
FWIW, I got "data area boundaries" from here file:///home/ruth/git_stuff/matplotlib/doc/build/html/api/spines_api.html#matplotlib.spines.Spine

Reply:
Fair enough because some of the spines do not have axis info on them.  I still found it an odd construction.